v2/api/keyvault/customizations/vault_extensions.go

```diff
@@ -33,6 +33,7 @@ import (
 
 var _ extensions.ARMResourceModifier = &VaultExtension{}
 
+//nolint:staticcheck // Underscores for enum values is our convention in ASO
```

Comment:
Is it possible to just disable this whole check (not all of staticcheck, just this one), rather than `nolint` everywhere?

Reply:
Will investigate and do a followup PR if I can.